backend/api/Services/InspectionFindingsService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Api.Database.Context;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace Api.Services
+{
+    public class InspectionFindingService(
+        ILogger<InspectionFindingService> logger,
+        IServiceScopeFactory serviceScopeFactory,
+        IHostApplicationLifetime hostApplicationLifetime,
+        IConfiguration configuration) : IHostedService
```

Comment:
Why does this class implement the "IHostedService" interface?

Reply:
Setting this as IHosted Service is recommended in dotnet for applications running for long time in the background, we have the same for Mqtt as well.

Reply:
Done.